docs/guides/06-introduction-to-animation.md

```diff
@@ -1,50 +1,47 @@
 ## Introduction to Animation
 
-Using CARTO VL you can make animated maps of points, lines, and polygons using the [`torque`](https://carto.com/developers/carto-vl/reference/#cartoexpressionstorque) expression.
-
-**Note:**
-While the expression name *torque* is inspired by another [CARTO technology](https://carto.com/torque/) for temporal mapping, the two are not equal. When creating temporal maps with CARTO VL, always refer to this documentation.
+Using CARTO VL you can make animated maps of points, lines, and polygons using the [`animation`](https://carto.com/developers/carto-vl/reference/#cartoexpressionsanimation) expression.
 
 ### Animation expression
 
 The general syntax for animating data in CARTO VL is:
 
 ```
-filter: torque(input, duration, fade(fadeIn, fadeOut))
+filter: animation(input, duration, fade(fadeIn, fadeOut))
 ```
 
 #### Filter
 
-The convention in CARTO VL is that `0` represents the boolean value `false`, (the absence of or an *off* state), while `1` represents `true` (the presence of or an *on* state). When the expression assigned to [`filter`](https://carto.com/developers/carto-vl/reference/#cartoexpressions) (`torque` in our case) has the value `0` for a given feature, that feature will be filtered-out and not be shown. If the value is `1`, the feature will be visible.
+The convention in CARTO VL is that `0` represents the boolean value `false`, (the absence of or an *off* state), while `1` represents `true` (the presence of or an *on* state). When the expression assigned to [`filter`](https://carto.com/developers/carto-vl/reference/#cartoexpressions) (`animation` in our case) has the value `0` for a given feature, that feature will be filtered-out and not be shown. If the value is `1`, the feature will be visible.
 
-#### Torque expression
+#### Animation expression
 
-The `torque` expression acts like a clock, generating cyclic values over a determinate period to match the values of its input; when a match occurs between the input and the internal cyclic value torque returns a `1` value and otherwise it returns a `0'.
+The `animation` expression acts like a clock, generating cyclic values over a determinate period to match the values of its input; when a match occurs between the input and the internal cyclic value animation returns a `1` value and otherwise it returns a `0'.
```

Comment:
Since we're finally calling *progress* to what we used to call simTime, I'd introudce the term here.
For example:
> The `animation` expression acts like a clock, generating cyclic progress values over a determinate period to match the values of its input; when a match occurs between the input and the progress value animation returns a `1` value and otherwise it returns a `0'.

Reply:
done!